spec/features/appointments_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+
+describe 'Appointments' do
+  let(:user) { FactoryBot.create(:user) }
+  let(:shop) { FactoryBot.create(:shop) }
+  let(:shop_slot1) { FactoryBot.create(:shop_slot, shop: shop) }
+  let(:shop_slot2) { FactoryBot.create(:shop_slot, shop: shop) }
+  let(:shop_slot3) { FactoryBot.create(:shop_slot, shop: shop) }
+  let(:shop_slot4) { FactoryBot.create(:shop_slot, shop: shop) }
+  let(:shop_slot5) { FactoryBot.create(:shop_slot, shop: shop) }
+  let(:shop_slot6) { FactoryBot.create(:shop_slot, shop: shop) }
+  let(:shop_slot7) { FactoryBot.create(:shop_slot, shop: shop) }
```

Comment:
this can be done in a sequence or in an each
or just add the slots in the factory. is much more easy